src/app.js

```diff
@@ -120,9 +121,27 @@ export function app() {
 		request: function() {},
 		/**
 		 * Spawn an app...
-		 * @returns {Promise}
+		 * @param {string} target Target appId. - No wildcards supported
+		 * @param {string} topic Topic.
+		 * @param {*=} data Data.
+		 * @returns {Object} Chainable
```

Comment:
I think you forgot the return :)

Reply:
actually, this is not chainable, it should return a promise or throw an error based on what the response is from the app!